### ISSUE TITLE
Adjust Color Scheme for Improved Readability

### ISSUE THREAD

**Issue by Entytaiment25** [datsfilipe/smooth-theme#15]
It's difficult to read text when brighter colors are used, and the color choices for the menu make it hard to read as well. Could you please adjust the colors for better readability?

Edit: Only active theme loaded

Reference Images:

![image](https://github.com/datsfilipe/smooth-theme/assets/64799287/bf0acb4a-6ced-493d-9286-f57ec46e716c)
![image](https://github.com/datsfilipe/smooth-theme/assets/64799287/5afdb62b-7337-4ffb-bc5a-61870a2163ea)
![image](https://github.com/datsfilipe/smooth-theme/assets/64799287/5adeb809-9cfe-4b28-9ed5-68414774ff6e)

**Comment by datsfilipe** [comment]
Yeah, originally for selected buttons the text color is more close to black. What happened is that Discord probably updated something that broke this.

Lately I'm very occupied with work, but I'll try to address this and some other problems in the next few days.

**Comment by datsfilipe** [comment]
also, is the last image for light theme? currently, Smooth theme has no support for light theme.

**Comment by mizu-048** [comment]
![image](https://github.com/datsfilipe/smooth-theme/assets/154871713/21d4aedf-553f-46ab-a0fe-b37897c50ae5)
![image](https://github.com/datsfilipe/smooth-theme/assets/154871713/a912dc0f-aee0-456e-a5c6-e9a87fca145a)
![image](https://github.com/datsfilipe/smooth-theme/assets/154871713/8321f54b-cb58-46e5-b0a4-7b8b7ab22687)
![image](https://github.com/datsfilipe/smooth-theme/assets/154871713/4cdbb38b-00d6-480b-89a1-59b14c01b223)
its everywhere...

**Comment by mizu-048** [comment]
![image](https://github.com/datsfilipe/smooth-theme/assets/154871713/9343980b-cbfe-493a-98ad-5f5b250f7dcc)
not relevant to this issue but there is a small tearing in the bottom

**Comment by datsfilipe** [comment]
hey @mizu-048, yes I've noticed those problems, they're not actually the colors fault, but `Discord` that updated the class names again (😢). I did not had time to look into it yet, but the problem is that the theme styles aren't being applied due to changes in `Discord` itself.

That said, I've plans to use wildcards for those class names when updating this theme, that way this same problem will probably not happen in a while. I'll update you here when I've anything.

**Comment by mizu-048** [comment]
alright, thank you, good luck with whatever you're busy with

**Comment by DaRandomCube** [comment]
8 months later...

**Comment by mizu-048** [comment]
@DaRandomCube 
might try jab at it later this month
for now i use [this](https://github.com/mizu-048/discord-css/blob/main/black-dark-theme.css)

**Comment by datsfilipe** [comment]
> 8 months later...

this happens when you get hired 🙃

anyway, I'll have some time this end of the year, so I'll try to fix this the way I said I would, so sorry I didn't have as much time as I'd like to work on the theme

**Comment by datsfilipe** [comment]
Just to update: started doing some work on it here: #16. Not sure when will be ready tho.

**Comment by datsfilipe** [comment]
PR merged, I believe most of the issues are covered, it's probably usable now. Not sure how it is about performance, so maybe use any optimization stuff you can.

Also, if some issue is still not fixed, like a popup or something, let's try to create separate issues for them, so they can be fixed one by one.
